script.js

```diff
@@ -0,0 +1,180 @@
+    document.querySelector('.home-page h1').style.color = '#ec534271';
+    closeMenu();
+  }
+  closeMenu();
+};
+// Show About Page
+
+const showAbout = function () {
+  document.querySelector('.text-container').classList.add('hidden');
+  document.querySelector('.LogoImageContainer').style.height = '36vmax';
+  document.querySelector('.text-containerAbout-page').classList.remove('hidden');
+  document.querySelector('.container p.display-1').style.textAlign = 'center';
+  document.querySelector('.container h1').style.textAlign = 'center';
+  document.getElementsByClassName('mainProgram')[0].classList.add('hidden');
+  document.getElementsByClassName('featuredSpeakers')[0].classList.add('hidden');
+  document.querySelector('.container h1').style.fontSize = '40px';
+  document.getElementsByClassName('about-page')[0].classList.remove('hidden');
+  document.getElementsByClassName('Partner')[0].classList.remove('hidden');
+  document.getElementsByClassName('dateAbout')[0].classList.remove('hidden');
+  document.getElementsByClassName('date')[0].classList.add('hidden');
+  document.querySelector('.home-page').style.marginBottom = '56px';
+  document.querySelector('.PartnerContainer').classList.remove('hidden');
+  document.querySelector('.finalLogo').style.backgroundColor = 'white';
+  document.querySelector('.container p.display-1').classList.remove('hidden');
+  document.querySelector('.home-page h1').style.color = '#ec534271';
+  document.querySelector('.container').style.alignItems = 'center';
+  document.querySelector('body').style.overflowY = 'initial';
+  document.querySelector('html').style.overflowY = 'initial';
+  if (window.screen.width >= 768) {
+    document.querySelector('.Partner h2').classList.add('hidden');
+    document.querySelector('.PartnerContainer').classList.add('hidden');
+    document.querySelector('.Partner').style.backgroundColor = '#252626';
+    document.querySelector('.finalLogo').style.backgroundColor = '#252626';
+    document.querySelector('.finalLogo').style.justifyContent = 'center';
+    document.querySelector('.finalLogo').style.marginTop = '10px';
+    document.querySelector('.finalLogoImage').style.marginRight = '10px';
+    document.querySelector('.finalLogoImage').style.marginBottom = '10px';
+    document.querySelector('.LogoImageContainer').style.height = '36vmax';
+    document.querySelector('.finalLogoImage').style.height = '8rem';
+    document.querySelector('.container h1').style.fontSize = '70px';
+    document.querySelector('.container').style.alignItems = 'center';
+    document.querySelector('.container p.display-1').classList.add('hidden');
+    document.querySelector('.container').style.marginTop = '40px';
+    document.querySelector('.container').style.marginLeft = '0';
+    document.querySelector('.home-page h1').style.color = '#ec5242';
+    document.querySelector('.buyTicket').style.color = '#ec5242';
+    document.querySelector('.container').style.alignItems = 'center';
```

Comment:
The same here

### [OPTIONAL]
- [ ] TO make this code DRY and KISS, I think we would improve it by putting all the styles in a CSS file, the javascript file should not contain styles, this will make our javascript clear and maintainable.